src/swap/types.ts

```diff
@@ -213,6 +213,10 @@ export type SwapContextType = {
   handleToggle: () => void;
   updateLifecycleStatus: (state: LifecycleStatusUpdate) => void; // A function to set the lifecycle status of the component
   to: SwapUnit;
+  isToastVisible?: boolean;
+  setIsToastVisible?: (visible: boolean) => void;
```

Comment:
I think a larger discussion, but how are we determining when state should live / be handled in the SwapContext vs. LifecycleStatus shared? cc @alessey

Reply:
I think the toasts are a case where we cannot rely on lifecycle status in case we want to display a toast longer than a lifecycle status persists.  Although I would prefer to encapsulate the toast state within its own component or a single toast state.

Reply:
re: https://github.com/coinbase/onchainkit/pull/1281#discussion_r1773436801